### ISSUE TITLE
About loss_robust implemented in trades.py

### ISSUE THREAD

**Issue by jh-jeong** [yaodongyu/TRADES#12]
I've changed `loss_robust` in `trades.py` to use cross entropy loss as the following, but then found that in that case the training fails to converge when `beta=6.0`. I suspect that this is because dCE(f(x), f(x'))/dw and dKL(f(x), f(x'))/dw is different as f(x') should be also a variable.  
As the paper claims that one should use classification-calibrated CE in TRADES to avoid the method to be the logit pairing, I wonder if is it ok to use KL instead of CE for `loss_robust`. Or have you considered some practices when `loss_robust` is implemented in CE, e.g. different `beta`?

```
def _cross_entropy(input, targets, reduction='mean'):
    targets_prob = F.softmax(targets, dim=1)
    xent = (-targets_prob * F.log_softmax(input, dim=1)).sum(1)
    if reduction == 'sum':
        return xent.sum()
    elif reduction == 'mean':
        return xent.mean()
    elif reduction == 'none':
        return xent
    else:
        raise NotImplementedError()

loss_robust = _cross_entropy(model(x_adv), model(x_natural), reduction='mean')
```

**Comment by yaodongyu** [comment]
Thanks for proposing this.

You could try to use different ```beta``` when you apply CE loss for ```loss_robust``` (you could try smaller ```beta```, like ```beta = 2.0```), and this will lead to similar performance (robust accuracy) compared with the KL one.
